### PR TITLE
Add auditd example with Auditbeat in kubernetes manifests

### PR DESCRIPTION
**Pull request by jsoriano**
Add an example configuration of the auditd module in the Auditbeat
reference manifest, including the processors needed for enrichement of
events.

For enrichement it makes use of #15947, included in 7.7.

### How to test?

* Run Auditbeat in kubernetes with the reference configuration.
* Exec a command inside a container, check that an event about this is collected.
* Try to access or write to a file without permissions in a container, check that an event about this is collected.
* Check that events collected about containers include the kubernetes metadata.

### CODE REVIEW

File: deploy/kubernetes/auditbeat-kubernetes.yaml [hunk 1]
```diff
@@ -16,6 +16,17 @@ data:
...
+          match_pids: ['process.pid']
+          include_fields: ['container.id']
+      - add_kubernetes_metadata:
+          host: ${NODE_NAME}
+          default_indexers.enabled: false
+          default_matchers.enabled: false
+          indexers:
+            - container:
+          matchers:
+            - fields.lookup_fields: ['container.id']
```
Comment (jsoriano) [inline deploy/kubernetes/auditbeat-kubernetes.yaml:29]: We could probably make these indexers and matchers the default ones for auditbeat.

Comment (exekias) [reply deploy/kubernetes/auditbeat-kubernetes.yaml:29]: ++ on doing this

Comment (jsoriano) [reply deploy/kubernetes/auditbeat-kubernetes.yaml:29]: Issue created for this: https://github.com/elastic/beats/issues/17432

**Comment by jsoriano** [comment]
Changelog entry added.

**Comment by jsoriano** [comment]
@andrewkroh @adriansr could any of you take a look to this PR? specially to the included auditd config. Thanks!

**Pull request merged**